WhiskyMan.Backend/WhiskyMan.Entities/User.cs

```diff
@@ -26,7 +26,6 @@ public class User
         [Required]
         [MaxLength(EntitiesConfig.EmailLength)]
         public string Email { get; set; }
-        [Required]
```

Comment:
Merge the master branch to your branch localy. There is some map test which won't be compiled due to this.